shell/src/main/java/org/apache/accumulo/shell/commands/DebugCommand.java

```diff
@@ -16,57 +16,31 @@
  */
 package org.apache.accumulo.shell.commands;
 
-import java.io.IOException;
-import java.util.Arrays;
-import java.util.Map;
-import java.util.Set;
-
-import org.apache.accumulo.core.util.BadArgumentException;
 import org.apache.accumulo.shell.Shell;
 import org.apache.accumulo.shell.Shell.Command;
-import org.apache.accumulo.shell.Token;
 import org.apache.commons.cli.CommandLine;
 
+/**
+ * @deprecated since 2.0; this command shouldn't be used; users should configure debug logging with
+ *             their log configuration file instead
+ */
+@Deprecated
 public class DebugCommand extends Command {
   @Override
-  public int execute(final String fullCommand, final CommandLine cl, final Shell shellState)
-      throws IOException {
-    if (cl.getArgs().length == 1) {
-      if (cl.getArgs()[0].equalsIgnoreCase("on")) {
-        Shell.setDebugging(true);
-      } else if (cl.getArgs()[0].equalsIgnoreCase("off")) {
-        Shell.setDebugging(false);
-      } else {
-        throw new BadArgumentException("Argument must be 'on' or 'off'", fullCommand,
-            fullCommand.indexOf(cl.getArgs()[0]));
-      }
-    } else if (cl.getArgs().length == 0) {
-      shellState.getReader().println(Shell.isDebuggingEnabled() ? "on" : "off");
-    } else {
-      shellState.printException(new IllegalArgumentException(
-          "Expected 0 or 1 argument. There were " + cl.getArgs().length + "."));
-      printHelp(shellState);
-      return 1;
-    }
-    return 0;
+  public int execute(final String fullCommand, final CommandLine cl, final Shell shellState) {
+    shellState.printException(new IllegalArgumentException("The debug command is deprecated; "
+        + "configure debug logging through your log configuration file instead."));
```

Comment:
Could add something like the following to this 

```
The debug command is deprecated;configure trace logging through your log configuration file instead.
This command used to enable trace logging for the org.apache.accumulo.core package.
If using the default configuration for the Apache Accumulo 2.0.0 binary distribution, modify $ACCUMULO_HOME/conf/log4j.properties to enable the trace logging.  If using other versions, 
, packaging, or configuration of Accumulo, the logging configuration may be elsewhere.  If you are unsure
how to enable trace logging please send a message to the accumulo user list.  
```

Reply:
I'm not sold on that specific proposed wording because:

1. I think that's a bit too verbose,
2. I don't think we can know what a "default configuration" is in any particular deployment, and this would be hard-coded, violating the "packagability" principle (in essence, not making assumptions in code about the deployment environment),
3. I'm not comfortable promising that users will get answers on the user list... and that should be the general default for any questions anyway, so it's probably redundant. It's also not a good recommendation if the user is getting their code repackaged from a vendor or packager.

Since I don't know how to address those concerns with succinct wording, and because I think it's better to assume user competence with logging configuration in their deployed environment, with the minimal wording I provided, at least as a starting point, I'll leave what I have for now. More can always be added later if it becomes obvious that it is necessary.

Reply:
> I think that's a bit too verbose,

I think its worth if it saves someone time.  My thinking behind this was, it would be nice to include the info I would tell someone of they were sitting next to me and asked how do I enable debugging in 2.0.0?

> I don't think we can know what a "default configuration" is in any particular deployment

That is why I suggested how to find it for a specific version and packaging. Again with the hope of saving someone time. There are a large number of changes in 2.0.0, each one potentially taking away time from users.

> I'm not comfortable promising that users will get answers on the user list..

Yeah, it probably makes sense to leave that out.

Reply:
> it's better to assume user competence with logging configuration in their deployed 

Also, I included some info about what the command use to do.  The fact the it used to enable TRACE for the accumulo.core package.  Not something I would expect someone who had used this command previously to know.